acceptance-tests/features/create_new_case.feature

```diff
@@ -4,12 +4,12 @@ Feature: Create case and deed helper page
   to have a page that will let them easily create a deed and a case for test purposes
 
 Scenario: Go to the create case page
-    Given I navigate to the case list page
-    When I click on the 'Create a new case' button
-    Then I am taken to the 'Create a new case' page
+    Given I navigate to the conveyancer frontend "/cases" page
+    When I click on the "Create a new case" link
```

Comment:
@nigelpain this is the link on the case list, which is a link called "Create a new case". The "Create case" button is on the next page.